examples/components/MNIST/traininsilo/spec.yaml

```diff
@@ -24,6 +24,11 @@ inputs:
     description: Iteration number
     default: 1
     optional: true
+  total_num_of_iterations:
```

Comment:
Nit: since "total_num_of_iterations" is only required for DP purpose, it might be slightly better to put this input in the same section as other dp-related args, e.g., after line 55 or line 70, and also add a doc string clarifying that those arg inputs (dp, dp-target_epsilon, dp_target_delta, dp_max_grad_norm, and total_num_of_iterations are defined for the only purpose of DP and can be ignored when users don't want to use DP.